kyo-reactive-streams/shared/src/main/scala/kyo/interop/flow/package.scala

```diff
@@ -0,0 +1,33 @@
+package kyo.interop
+
+import java.util.concurrent.Flow.*
+import kyo.*
+import kyo.interop.reactivestreams
+import kyo.kernel.Boundary
+import org.reactivestreams.FlowAdapters
+
+package object flow:
+    inline def fromPublisher[T](
+        publisher: Publisher[T],
+        bufferSize: Int
+    )(
+        using
+        Frame,
+        Tag[T]
+    ): Stream[T, Async] < IO = reactivestreams.fromPublisher(FlowAdapters.toPublisher(publisher), bufferSize)
+
+    def subscribeToStream[T, Ctx](
+        stream: Stream[T, Ctx],
+        subscriber: Subscriber[? >: T]
+    )(
+        using
+        Boundary[Ctx, IO],
+        Frame,
+        Tag[T]
```

Comment:
It'd be ideal to take `Tag[Emit[Chunk[T]]]` evidences directly otherwise the tag macro has to dynamically compose tags, which requires allocation.